amqp/schema_channel.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Channel Schema",
+  "type": "object",
+  "properties": {
+    "is": {
+      "type": "string",
+      "enum": ["queue", "routingKey"]
+    },
+    "exchange": {
+      "type": "object",
```

Comment:
I couldn't figure out how to add Map type. should I change `object` to `any`?

Reply:
`object` is exactly the way. You can always manually test schema against real object (taken from example) using https://www.jsonschemavalidator.net/